src/utils/navigation.ts

```diff
@@ -68,6 +68,11 @@ export const NAVIGATION: Navigation = {
   [config.UPDATE_DUE_DILIGENCE_OVERSEAS_ENTITY_URL]: {
     currentPage: config.UPDATE_DUE_DILIGENCE_OVERSEAS_ENTITY_PAGE,
     previousPage: () => config.WHO_IS_MAKING_UPDATE_PAGE,
+    nextPage: [config.UPDATE_REVIEW_OVERSEAS_ENTITY_INFORMATION_URL]
+  },
+  [config.UPDATE_REVIEW_OVERSEAS_ENTITY_INFORMATION_URL]: {
+    currentPage: config.UPDATE_REVIEW_OVERSEAS_ENTITY_INFORMATION_PAGE,
+    previousPage: () => config.WHO_IS_MAKING_UPDATE_PAGE,
```

Comment:
Please check the entries for page on line 78
[config.OVERSEAS_ENTITY_REVIEW_URL]: {
}
Some page must now have this new one as previous

Reply:
Actually correct to skip back over the info screen but should be confirmed with BA.

Reply:
Checked with BA and previous/back button for OE review page needs to be new info page.